doc/manual/faq.rst

```diff
@@ -1,110 +1,169 @@
-.. Copyright (C) 2014, 2015 Robert Jordens <jordens@gmail.com>
+FAQ (How do I...)
+=================
 
-FAQ
-===
+use this documentation?
+-----------------------
 
-How do I ...
-------------
+The content of this manual is arranged in rough reading order. If you start at the beginning and make your way through section by section, you should form a pretty good idea of how ARTIQ works and how to use it. Otherwise:
+
+**If you are just starting out,** and would like to get ARTIQ set up on your computer and your core device, start with :doc:`installing`, :doc:`flashing`, and :doc:`configuring`, in that order.
+
+**If you have a working ARTIQ setup** (or someone else has set it up for you), start with the tutorials: read :doc:`rtio`, then progress to :doc:`getting_started_core` and :doc:`getting_started_mgmt`. If your system is in a DRTIO configuration, :doc:`DRTIO and subkernels <using_drtio_subkernels>` will also be helpful.
+
+Pages like :doc:`management_system` and :doc:`core_device` describe **specific components of the ARTIQ ecosystem** in more detail. If you want to understand more about device and dataset databases, for example, read the Environment page; if you want to understand the ARTIQ Python dialect and everything it does or does not support, read the Compiler page.
+
+Reference pages, like :doc:`main_frontend_tools` and :doc:`core_drivers_reference`, contain the detailed documentation of the individual methods and command-line tools ARTIQ provides. They are heavily interlinked throughout the rest of the documentation: whenever a method, tool, or exception is mentioned by name, like :class:`~artiq.frontend.artiq_run`, :meth:`~artiq.language.core.now_mu`, or :exc:`~artiq.coredevice.exceptions.RTIOUnderflow`, it can normally be clicked on to directly access the reference material.
+
+.. _build-documentation:
+
+build this documentation?
+-------------------------
+
+To generate this manual from source, you can use ``nix build`` directives, for example: ::
+
+    $ nix build git+https://github.com/m-labs/artiq.git\?ref=release-[number]#artiq-manual-html
+
+Substitute ``artiq-manual-pdf`` to get the LaTeX PDF version. The results will be in ``result``.
+
+The manual is written in `reStructured Text <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_; you can find the source files in the ARTIQ repository under ``doc/manual``. If you spot a mistake, a typo, or something that's out of date or missing -- in particular, if you want to add something to this FAQ -- feel free to clone the repository, edit the RST files, and make a pull request with your version of an improvement. The second best thing is to open an issue to make M-Labs aware of the problem.
```

Comment:
well, just clone is not enough. They'll need to fork it first (in case user is not much familiar with git and common practices)